src/main/java/net/smartcosmos/cluster/userdetails/resource/AuthenticationResource.java

```diff
@@ -9,32 +9,31 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
 
 import net.smartcosmos.annotation.SmartCosmosRdao;
-import net.smartcosmos.cluster.userdetails.dto.RestAuthenticateRequest;
-import net.smartcosmos.cluster.userdetails.service.AuthenticationService;
+import net.smartcosmos.cluster.userdetails.domain.AuthenticateUserRequest;
+import net.smartcosmos.cluster.userdetails.service.AuthenticateUserService;
 
 import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
 
-@SmartCosmosRdao
-@RestController
 @Slf4j
+@SmartCosmosRdao
 public class AuthenticationResource {
 
-    private final AuthenticationService authenticationService;
+    private final AuthenticateUserService service;
 
     @Autowired
-    public AuthenticationResource(AuthenticationService authenticationService) {
+    public AuthenticationResource(AuthenticateUserService authenticateUserService) {
 
-        this.authenticationService = authenticationService;
+        this.service = authenticateUserService;
```

Comment:
technically if the variable names don't match then `this` is not required to clarify scope and becomes superfluous.  I am not a fan of `this.`